DQMOffline/Muon/python/EfficencyPlotter_cfi.py

```diff
@@ -100,10 +100,55 @@
                                           etaMax = cms.double(2.5),
                                           phiMax = cms.double(3.2),
                                           ptMax  = cms.double(100),
-                                          vtxBin = cms.int32(10),
+                                          vtxBin = cms.int32(30),
                                           vtxMin = cms.double(0.5),
-                                          vtxMax = cms.double(40.5),
+                                          vtxMax = cms.double(149.5),
                                           MuonID = cms.string("Tight")
                                           )
 
 
+effPlotter=cms.Sequence(effPlotterLoose*effPlotterMedium*effPlotterTight)
+effPlotter_miniAOD=cms.Sequence(effPlotterLooseMiniAOD*effPlotterMediumMiniAOD*effPlotterTightMiniAOD)   
+
+effPlotterLoose_Phase2=effPlotterLoose.clone()
+effPlotterLoose_Phase2.vtxBin=20
+effPlotterLoose_Phase2.vtxMin=149.5
+effPlotterLoose_Phase2.vtxMax=249.5
```

Comment:
```suggestion
effPlotterLoose_Phase2=effPlotterLoose.clone(
  vtxBin=20
  vtxMin=149.5
  vtxMax=249.5
)
```
And the same for the other cases below